Add Mirai. Tournament 2020 (#3947)
ppy/osu-wiki: wiki/Tournaments/Mirai/2020/en.md

---
tags:
  - Mirai2020
---

# Mirai. Tournament 2020

> ![Mirai2020 Logo](img/logo.png)

Reviewer: ```suggestion
![Mirai2020 logo](img/logo.png)
```

The **Mirai. Tournament 2020** (***Mirai2020***) is a worldwide double-eliminination 2v2 osu! tournament hosted by ![][flag_NL] [Mirai Subject](https://osu.ppy.sh/users/5639709). It is the fourth installment of the Mirai. Tournament.

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2020-04-06/2020-04-19 |
| Qualifiers | 2020-05-02 |
| Round of 32 | 2020-05-08 |
| Round of 16/Losers Round 1 & 2 | 2020-05-16 |
| Quarterfinals/Losers Round 2 & 3 | 2020-05-22 |
| Semifinals | 2020-05-30 |
| Finals - Week 1 | 2020-06-06 |
| Finals - Week 2 | 2020-06-23 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 4 months of osu!supporter tag per team member, unique profile badge (pending) |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 1 month of osu!supporter tag per team member |

## Organisation

> The Mirai. Tournament 2020 is run by various community members by distributing the multitude of tasks into various fields of responsibility.

Reviewer: ```suggestion
The Mirai. Tournament 2020 is run by various community members.
```
should be enough

| im pepa gpig | -1 | **0** | **8 min tech map** | *win by default* |
| LowAcc Players | -1 | **0** | **Bokeee** | *win by default* |
| **Nanahira Copypasta** | **0** | -1 | Nice Nice Nice | *win by default* |
| **HappyHuskyBubbles** | **0** | -1 | FastButNotFurious | *win by default* |

### Qualifiers

| Seed | Team |
| :-- | :-- |
| 1 | Airline Food |
| 2 | surprised_pikachu |
| 3 | Tempe |
| 4 | How2NoMod |
| 5 | Retirement Home |
| 6 | Uprankers |
| 7 | kamiguozi |
| 8 | shaky hand |
| 9 | Norank MaMa <3 U |
| 10 | FastButNotFurious |
| 11 | Baka gang. |
| 12 | cherry gum v4 |
| 13 | WhamDabbFTW |
| 14 | The |
| 15 | Rip OsumeWolf |
| 16 | 8 min tech map |
| 17 | Makowy |
| 18 | Your Average Team |
| 19 | bruh moment |
| 20 | Nice Nice Nice |
| 21 | WorldWide Bootleg |
| 22 | Rusty Cucumber |
| 23 | Salted_Fish |
| 24 | Bokeee |
| 25 | LowAcc Players |
| 26 | HappyHuskyBubbles |
| 27 | bee movie 2 2 |
| 28 | Vanish |
| 29 | Nanahira Copypasta |
| 30 | Baeguette |
| 31 | Team Sliderbreak |
| 32 | im pepa gpig |

---

## Ruleset

The ruleset for this tournament is: Scorev2, Team VS in osu! standard.

> ### Tournament Rules

Reviewer: ```suggestion
### Tournament rules
```

1. The Discord server rules apply. Failure to comply will result in the disqualification of you and/or your team depending on the severity and the people involved.
2. The rank range for this tournament is 105.000 - 45.000.
3. If during this tournament you get derank under 105.000 or overrank above 45.000 you will not be allowed to play anymore in your team until you've deranked enough to be in range again.
4. Grieving, trolling, and other actions that may hinder a match are prohibited.
5. Sportsmanship is encouraged. Don’t be mean towards other players.
6. Cheating is not allowed (obviously)
7. Follow the instructions your assigned referee has given. For example: do not hop around in player slots and waste the referee's time.

> ### Tournament Registration

Reviewer: ```suggestion
### Tournament registration
```

All the players are required to register on [mirai.gg](https://mirai.gg/). On the website, players login using their osu! account and connect it to their discord account. Once all the players of a team have joined the team captain can create the team on the website. The BWS rank verification of all the players takes place once the captain has finalized the team.

Another BWS rank verification takes place on the day of the sign up deadline to ensure all the players are within rank range.

If a player changes their osu! name during the course of the tournament, they only need to login to the website for the change to be propagated across the tournament's systems.

After the sign up deadline, a list of all the players will be sent to the osu! tournament staff for screening. Players that do not pass screening will not be allowed to participate in the tournament.

> ### Stage Instructions

Reviewer: ```suggestion
### Stage instructions
```

The length for each stage is as follows, these include their respective Losers Rounds:

- RO32 & RO16: BO9
- QF and SF: BO11
- (Grand) Finals: BO13

> ### Match Instructions

Reviewer: ```suggestion
### Match instructions
```

1. The assigned referee will invite the team captain 10-15 minutes before the match starts
2. The team captain will invite their teammates to the room.
3. When the referee gives permission the team captains will !roll

> 
> The team that rolls the highest gets to pick whether they want to ban first and pick first.
> (So highest roller decides pick and ban order)
> They can choose if they want to pick and ban second if they want to.
> Players can choose to skip their warmup.
> In the rare event that both teams roll the same number, they will roll again.
> 
> Each team gets to pick one warmup map. They can decide who or can use the roll results.

Reviewer: ```suggestion
4. The team that rolls the highest gets to pick whether they want to ban first and pick first.
5. Highest roller decides pick and ban order. They can choose if they want to pick and ban second if they want to.
6. Players can choose to skip their warmup.
7. In the rare event that both teams roll the same number, they will roll again.
8. Each team gets to pick one warmup map. They can decide who or can use the roll results.

```

- Pending or graveyarded maps are prohibited.
- Loved, qualified, and ranked maps within the above criteria are allowed

After the warm-up, the banning phase starts.

If the highest roller chooses to ban first the procedure will be as follows:

- High roll: 1 ban
- Low roll: 2 bans
- High roll: 1 ban

If the highest roller chooses to ban second it will be as follows:

- Low roll: 1 ban
- High roll: 2 bans
- Low roll: 1 ban

Modpools cannot be fully banned. At least one map has to be available in each modpool

After the ban phase, the team will play maps with alternating picks until one team wins enough points to win a match.

1. The tiebreaker cannot be picked in a normal game scenario.
2. ForceMod maps are limited to: HD, HR, FL, EZ. You can stack these mods for a higher risk-higher reward scenario. With ForceMod at least one player has to pick a mod.
3. If both of the teams draw the map, then that map will not be counted towards the final score. However, it will be recorded for statistical purposes.
4. The tiebreaker is Free Mod, which means you can pick the mods from rule B or go NoMod

> ### Rescheduling Instructions

Reviewer: ```suggestion
### Rescheduling instructions
```

---
tags:
  - Mirai2020
---

# Mirai. Tournament 2020

![Mirai2020 Logo](img/logo.png)

> The **Mirai. Tournament 2020** (**_Mirai2020_**) is a worldwide 2v2 double eliminination osu! tournament hosted by ![][flag_nl] [Mirai Subject](https://osu.ppy.sh/users/5639709). It is the 4th installment of the Mirai. Tournament.

Reviewer 2: ```suggestion
The **Mirai. Tournament 2020** (***Mirai2020***) is a worldwide double-eliminination 2v2 osu! tournament hosted by ![][flag_NL] [Mirai Subject](https://osu.ppy.sh/users/5639709). It is the fourth installment of the Mirai. Tournament.
```

## Tournament Schedule

|                              Event | Timestamp             |

> | ---------------------------------: | :-------------------- |

Reviewer 2: ```suggestion
| --: | :-- |
```
Please refer to this guideline for table formatting

`:--` - Left Align
`:-:` - Middle Align
`--:` - Right Align

And only add 1 whitespace in the beginning and the end of the structure.
If it's an empty table, tolerance 2 paddings.

> |                 Registration Phase | 2020-04-06/2020-04-19 |

Reviewer 2: ```suggestion
| Registration phase | 2020-04-06/2020-04-19 |
```

> |                         Qualifiers | 2020-05-02            |

Reviewer 2: ```suggestion
| Qualifiers | 2020-05-02 |
```

> |                        Round of 32 | 2020-05-08            |

Reviewer 2: ```suggestion
| Round of 32 | 2020-05-08 |
```

> |   Round of 16 / Losers Round 1 & 2 | 2020-05-16            |

Reviewer 2: ```suggestion
| Round of 16/Losers Round 1 & 2 | 2020-05-16 |
```

> | Quarterfinals / Losers Round 2 & 3 | 2020-05-22            |

Reviewer 2: ```suggestion
| Quarterfinals/Losers Round 2 & 3 | 2020-05-22 |
```

> |                         Semifinals | 2020-05-30            |

Reviewer 2: ```suggestion
| Semifinals | 2020-05-30 |
```

> |                    Finals - Week 1 | 2020-06-06            |

Reviewer 2: ```suggestion
| Finals - Week 1 | 2020-06-06 |
```

> |                    Finals - Week 2 | 2020-06-23            |

Reviewer 2: ```suggestion
| Finals - Week 2 | 2020-06-23 |
```

## Prizes

> In every tournament, people conquering a place on the podium are eligible to receive unique prizes created specifically for Mirai. Tournament 2020 winners. It is possible that these items change with every installment of the Mirai. Tournament.

Reviewer 2: ```suggestion

```
You can delete this

> |                          Placing                           | Prize(s)                                                               |

Reviewer 2: ```suggestion
| Placing | Prize(s) |
```

> | :--------------------------------------------------------: | :--------------------------------------------------------------------- |

Reviewer 2: ```suggestion
| :-: | :-- |
```

> |   ![Gold Crown](/wiki/shared/crown-gold.png "1st place")   | 4 months of osu! supporter per team member & a profile badge (pending) |

Reviewer 2: ```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 4 months of osu!supporter tag per team member, unique profile badge (pending) |
```

> | ![Silver Crown](/wiki/shared/crown-silver.png "2nd place") | 1 month of osu! supporter per team member                              |

Reviewer 2: ```suggestion
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 1 month of osu!supporter tag per team member |
```

## Organisation

The Mirai. Tournament 2020 is run by various community members by distributing the multitude of tasks into various fields of responsibility.

> | Position     | Member(s)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |

Reviewer 2: ```suggestion
| Position | Member(s) |
```

> | :----------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

Reviewer 2: ```suggestion
| :-- | :-- |
```

> | Manager      | ![][flag_ph] [Deadpulse](https://osu.ppy.sh/users/8575527), ![][flag_ph] [kristoffer09](https://osu.ppy.sh/users/3474344)                                                                                                                                                                                                                                                                                                                                                                                                                                    |

Reviewer 2: ```suggestion
| Manager | ![][flag_PH] [Deadpulse](https://osu.ppy.sh/users/8575527), ![][flag_PH] [kristoffer09](https://osu.ppy.sh/users/3474344) |
```

> | Map Selector | ![][flag_gb] [AlexDark69](https://osu.ppy.sh/users/5146647), ![][flag_fr] [Kaeldori](https://osu.ppy.sh/users/962519), ![][flag_gb] [NoSpace](https://osu.ppy.sh/users/10585285), ![][flag_gb] [SacretSword](https://osu.ppy.sh/users/9821402), ![][flag_de] [Windows 10 Pro](https://osu.ppy.sh/users/7486592), ![][flag_sg] [Zephyrlox](https://osu.ppy.sh/users/6436945)                                                                                                                                                                                  |

Reviewer 2: ```suggestion
| Map selector | ![][flag_GB] [AlexDark69](https://osu.ppy.sh/users/5146647), ![][flag_FR] [Kaeldori](https://osu.ppy.sh/users/962519), ![][flag_GB] [NoSpace](https://osu.ppy.sh/users/10585285), ![][flag_GB] [SacretSword](https://osu.ppy.sh/users/9821402), ![][flag_DE] [Windows 10 Pro](https://osu.ppy.sh/users/7486592), ![][flag_SG] [Zephyrlox](https://osu.ppy.sh/users/6436945) |
```

> | Streamer     | ![][flag_nl] [Mirai Subject](https://osu.ppy.sh/users/5639709)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |

Reviewer 2: ```suggestion
| Streamer | ![][flag_NL] [Mirai Subject](https://osu.ppy.sh/users/5639709) |
```

> | Commentator  | ![][flag_gb] [AlexDark69](https://osu.ppy.sh/users/5146647), ![][flag_ph] [kristoffer09](https://osu.ppy.sh/users/3474344), ![][flag_tr] [oralekin](https://osu.ppy.sh/users/7631823), ![][flag_de] [Windows 10 Pro](https://osu.ppy.sh/users/7486592)                                                                                                                                                                                                                                                                                                       |

Reviewer 2: ```suggestion
| Commentator | ![][flag_GB] [AlexDark69](https://osu.ppy.sh/users/5146647), ![][flag_PH] [kristoffer09](https://osu.ppy.sh/users/3474344), ![][flag_TR] [oralekin](https://osu.ppy.sh/users/7631823), ![][flag_DE] [Windows 10 Pro](https://osu.ppy.sh/users/7486592) |
```

> | Designer     | ![][flag_nl] [Mirai Subject](https://osu.ppy.sh/users/5639709)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |

Reviewer 2: ```suggestion
| Designer | ![][flag_NL] [Mirai Subject](https://osu.ppy.sh/users/5639709) |
```

> | Referee      | ![][flag_gb] [AlexDark69](https://osu.ppy.sh/users/5146647), ![][flag_de] [GDLenny](https://osu.ppy.sh/users/8406711), ![][flag_de] [Inteleon](https://osu.ppy.sh/users/9665028), ![][flag_tr] [oralekin](https://osu.ppy.sh/users/7631823), ![][flag_nl] [Mirai Subject](https://osu.ppy.sh/users/5639709), ![][flag_vn] [rock-on](https://osu.ppy.sh/users/9676089), ![][flag_ph] [Simon Yuusuke](https://osu.ppy.sh/users/7336035), ![][flag_ca] [Takiaz](https://osu.ppy.sh/users/11177233), ![][flag_ca] [TheHunter1](https://osu.ppy.sh/users/6496016) |

Reviewer 2: ```suggestion
| Referee | ![][flag_GB] [AlexDark69](https://osu.ppy.sh/users/5146647), ![][flag_DE] [GDLenny](https://osu.ppy.sh/users/8406711), ![][flag_DE] [Inteleon](https://osu.ppy.sh/users/9665028), ![][flag_TR] [oralekin](https://osu.ppy.sh/users/7631823), ![][flag_NL] [Mirai Subject](https://osu.ppy.sh/users/5639709), ![][flag_VN] [rock-on](https://osu.ppy.sh/users/9676089), ![][flag_PH] [Simon Yuusuke](https://osu.ppy.sh/users/7336035), ![][flag_CA] [Takiaz](https://osu.ppy.sh/users/11177233), ![][flag_CA] [TheHunter1](https://osu.ppy.sh/users/6496016) |
```

> | Statistician | ![][flag_ph] [kristoffer09](https://osu.ppy.sh/users/3474344)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |

Reviewer 2: ```suggestion
| Statistician | ![][flag_PH] [kristoffer09](https://osu.ppy.sh/users/3474344) |
```

## Links

- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1047235)
- [Challonge brackets](https://challonge.com/Mirai2020)

> - [Mirai. Community Discord Server](https://discord.gg/cSfhfm2)

Reviewer 2: ```suggestion
- [Mirai. Community Discord server](https://discord.gg/cSfhfm2)
```

- [Livestream](https://www.twitch.tv/MiraiSubject)

## Participants

> |          Team          | Members                                                                                                                                                                                            |

Reviewer 2: ```suggestion
| Team | Members |
```

> | :--------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

Reviewer 2: ```suggestion
| :-: | :-- |
```

> |   **8 min tech map**   | [poboi06](https://osu.ppy.sh/users/12836907), [SirPinky](https://osu.ppy.sh/users/12948136), [meapii](https://osu.ppy.sh/users/12643092), [His](https://osu.ppy.sh/users/5433421)                  |

Reviewer 2: ```suggestion
| **8 min tech map** | [poboi06](https://osu.ppy.sh/users/12836907), [SirPinky](https://osu.ppy.sh/users/12948136), [meapii](https://osu.ppy.sh/users/12643092), [His](https://osu.ppy.sh/users/5433421) |
```
Remove unnecessary whitespaces. You only need one padding in the end and beginning of a content.
And if you can, stick player's flag

|   **cherry gum v4**    | [steve04](https://osu.ppy.sh/users/10852911), [DanX](https://osu.ppy.sh/users/10542408), [ABKirito](https://osu.ppy.sh/users/9093149)                                                              |
|    **im pepa gpig**    | [Nova Cobalt](https://osu.ppy.sh/users/4272776), [fw8te](https://osu.ppy.sh/users/14541203), [CrappySalami](https://osu.ppy.sh/users/8955826)                                                      |
|     **kamiguozi**      | [Guozi on Osu](https://osu.ppy.sh/users/8692802), [na-gi](https://osu.ppy.sh/users/11355787), [- Rainbow -](https://osu.ppy.sh/users/8742486)                                                      |
|     **shaky hand**     | [Fallen_HK](https://osu.ppy.sh/users/13981456), [Himman](https://osu.ppy.sh/users/12665894), [McFuddyDuddy](https://osu.ppy.sh/users/8825522), [Player01](https://osu.ppy.sh/users/10194478)       |
| **surprised_pikachu**  | [-Secured-](https://osu.ppy.sh/users/14648196), [Pokeinfernape](https://osu.ppy.sh/users/11412507), [7heGh0st](https://osu.ppy.sh/users/14256252), [Emfyr](https://osu.ppy.sh/users/13445383)      |

---

> ![Mirai2020 Bracket](img/bracket.png)

Reviewer 2: ```suggestion
![Mirai2020 bracket](img/bracket.png)
```

  - [Pendulum - The Vulture \[Insane\]](https://osu.ppy.sh/beatmapsets/24163#osu/82249)
- ForceMod
  - [yuikonnu - Kakushigoto \[Extra\]](https://osu.ppy.sh/beatmapsets/504245#osu/1085026)
  - [Shoichiro Sakamoto - Eye of Aeon \[Aeon\]](https://osu.ppy.sh/beatmapsets/26528#osu/89362)
  - [paraoka - boot \[sammish's Insane\]](https://osu.ppy.sh/beatmapsets/464095#osu/2084879)
- Tiebreaker
  - **[Our Stolen Theory - United (L.A.O.S Remix) \[Infinity\]](https://osu.ppy.sh/beatmapsets/237768#osu/550235)**

### Finals

**[Download the mappack here! (124 MB)](https://cdn.mirai.gg/mirai-2020/pool/mt-2020-finals-F16F65C5.zip)**

- NoMod
  - [BABYMETAL - YAVA! \[Extra\]](https://osu.ppy.sh/beatmapsets/476612#osu/1069950)
  - [Memme - Avalanche \[Xilver's Extra\]](https://osu.ppy.sh/beatmapsets/340048#osu/1009829)
  - [Kirin feat. Aoki Lapis - Not Found \[404\]](https://osu.ppy.sh/beatmapsets/560326#osu/1184958)
  - [Tomoe Tamiyasu - Yamato Nadeshiko Hanakotoba \[Extra\]](https://osu.ppy.sh/beatmapsets/748589#osu/1612947)
  - [Hakuryu - Genesis At Oasis (MOONLiGHT Mix) \[Vell's Insane\]](https://osu.ppy.sh/beatmapsets/500816#osu/1088043)

>   - [yak_won - Sinus ~Secret Heart~ \[Extra\]](https://osu.ppy.sh/beatmapsets/736694#osu/1569283)

Reviewer 2: ```suggestion
  - [yak\_won - Sinus \~Secret Heart\~ \[Extra\]](https://osu.ppy.sh/beatmapsets/736694#osu/1569283)
```

- Hidden
  - [CYTOKINE - Koyoi no Kage wo kiritoru Kotonohano (REDALiCE Remix) \[Lunatic\]](https://osu.ppy.sh/beatmapsets/44710#osu/139981)
  - [sakuzyo - Aldebaran \[Expert\]](https://osu.ppy.sh/beatmapsets/607948#osu/1283985)
  - [USAO - Night sky \[Another\]](https://osu.ppy.sh/beatmapsets/376397#osu/824622)
- HardRock
  - [REDALiCE - Volar \[Insane\]](https://osu.ppy.sh/beatmapsets/32156#osu/105311)
  - [dj TAKA meets DJ YOSHITAKA ft.guit.good-cool - Elemental Creation -GITADO ROCK ver.- \[Extreme\]](https://osu.ppy.sh/beatmapsets/119447#osu/307289)
  - [nano - INFINITY ZERO \[Insane\]](https://osu.ppy.sh/beatmapsets/230022#osu/535188)
- DoubleTime

>   - [Ray - lull ~Soshite Bokura wa~ \[Insane\]](https://osu.ppy.sh/beatmapsets/172450#osu/438061)

Reviewer 2: ```suggestion
  - [Ray - lull \~Soshite Bokura wa\~ \[Insane\]](https://osu.ppy.sh/beatmapsets/172450#osu/438061)
```

  - [Kalon. - Fumetsu no Gekijou \[Nikakis' Hard\]](https://osu.ppy.sh/beatmapsets/848312#osu/1821927)
  - [USAO - FREEDOM (Extended Mix) \[Hard\]](https://osu.ppy.sh/beatmapsets/749304#osu/1707909)
  - [Razihel & Virtual Riot - One For All, All For One \[Celsius' Insane\]](https://osu.ppy.sh/beatmapsets/275655#osu/687524)
- ForceMod
  - [Yuaru - Asu no Yozora Shoukaihan \[Linada's Insane\]](https://osu.ppy.sh/beatmapsets/384772#osu/854616)
  - [44teru-k - F.I \[Gold\]](https://osu.ppy.sh/beatmapsets/42225#osu/136411)

>   - [Akita Neru - Gotya Gotya Uruse~! \[Insane\]](https://osu.ppy.sh/beatmapsets/22182#osu/76638)

Reviewer 2: ```suggestion
  - [Akita Neru - Gotya Gotya Uruse\~! \[Insane\]](https://osu.ppy.sh/beatmapsets/22182#osu/76638)
```

  - [zts - worldenddominator \[Hard\]](https://osu.ppy.sh/beatmapsets/897884#osu/1886520)
  - [a\*ru - Kizuato \[Hard\]](https://osu.ppy.sh/beatmapsets/569843#osu/1241841)
  - [Rostik - Liquid (Paul Rosenthal Remix) \[Insane\]](https://osu.ppy.sh/beatmapsets/123593#osu/315552)
- ForceMod
  - [Shimotsuki Haruka - Akahitoha \[Rio & z1's Another\]](https://osu.ppy.sh/beatmapsets/974417#osu/2261415)
  - [IOSYS - Chanteikku Sanyousei no Itazura Daisensou \[Crazy Jay\]](https://osu.ppy.sh/beatmapsets/24448#osu/91462)
  - [P\*Light - SAY BAY \[Pata-Mon's Insane\]](https://osu.ppy.sh/beatmapsets/750316#osu/1588830)
- Tiebreaker
  - **[FELT - Vagueness & JOURNEY \[Collab Rhapsody\]](https://osu.ppy.sh/beatmapsets/909888#osu/1898745)**

### Quarterfinals

**[Download the mappack here! (136 MB)](https://cdn.mirai.gg/mirai-2020/pool/mt-2020-qf-1447801C.zip)**

- NoMod
  - [Reol - Logic Agent \[Lost\]](https://osu.ppy.sh/beatmapsets/578814#osu/1283956)
  - [Mary - Artificial Rose \[Another\]](https://osu.ppy.sh/beatmapsets/99434#osu/264439)
  - [lapix - Horizon Blue feat. Kanata.N \[Reform's Insane\]](https://osu.ppy.sh/beatmapsets/974832#osu/2062581)

>   - [LhoU - popotnik ~ The Traveller of Ljubljana \[Exotic\]](https://osu.ppy.sh/beatmapsets/712388#osu/1505919)

Reviewer 2: ```suggestion
  - [LhoU - popotnik \~ The Traveller of Ljubljana \[Exotic\]](https://osu.ppy.sh/beatmapsets/712388#osu/1505919)
```

  - [Lite Show Magic - TRICKL4SH 220 \[LCFC's INS4NE\]](https://osu.ppy.sh/beatmapsets/616696#osu/1308410)
  - [ONE OK ROCK - Re:make \[AngelHoney\]](https://osu.ppy.sh/beatmapsets/64001#osu/197139)
- Hidden
  - [seiya-murai feat.ALT - Sumidagawa Karenka \[Extreme\]](https://osu.ppy.sh/beatmapsets/28796#osu/95870)
  - [Function Phantom - Variable \[Kibbleru's Insane\]](https://osu.ppy.sh/beatmapsets/751774#osu/1582596)

>   - [Project Grimoire - Caliburne ~Story of the Legendary sword~ \[iyasine's Insane\]](https://osu.ppy.sh/beatmapsets/335665#osu/747240)

Reviewer 2: ```suggestion
  - [Project Grimoire - Caliburne \~Story of the Legendary sword\~ \[iyasine's Insane\]](https://osu.ppy.sh/beatmapsets/335665#osu/747240)
```

- HardRock
  - [Foreground Eclipse - You Can't Explain Anything Without The Word 'Destruction' \[Akitoforce's Hard\]](https://osu.ppy.sh/beatmapsets/942330#osu/1968524)
  - [SHW - Ikusa JAPAN (shw.in) \[Ikusa!\]](https://osu.ppy.sh/beatmapsets/46660#osu/145165)
- DoubleTime
  - [Becky - Less Than Three (Ricardo Autobahn Remix) \[Hard\]](https://osu.ppy.sh/beatmapsets/1078502#osu/2258857)
  - [Fox Stevenson - Turn It Up (Higher) \[Exquisite\]](https://osu.ppy.sh/beatmapsets/115291#osu/298035)
  - [Star Driver - 5 Centimeters Per Second \[Hard\]](https://osu.ppy.sh/beatmapsets/239787#osu/556326)
- ForceMod
  - [Days N' Daze - Misanthropic Drunken Loner \[Insane\]](https://osu.ppy.sh/beatmapsets/442587#osu/959058)
  - [Lon - MATRYOSHKA \[Tari's Insane\]](https://osu.ppy.sh/beatmapsets/109185#osu/315941)
- Tiebreaker
  - **[Reol - Minazoko Yuuhodou \[Ripples\]](https://osu.ppy.sh/beatmapsets/399810#osu/870151)**

### Round of 32

**[Download the mappack here! (101 MB)](https://cdn.mirai.gg/mirai-2020/pool/mt-2020-ro32-C86401FF.zip)**

- NoMod
  - [sakuzyo feat. Hatsune Miku - ChaiN De/structioN \[Loneliness\]](https://osu.ppy.sh/beatmapsets/49071#osu/154165)

>   - [ZUN - Kobito of the Shining Needle ~ Little Princess \[Lunatic\]](https://osu.ppy.sh/beatmapsets/362989#osu/827019)

Reviewer 2: ```suggestion
  - [ZUN - Kobito of the Shining Needle \~ Little Princess \[Lunatic\]](https://osu.ppy.sh/beatmapsets/362989#osu/827019)
```

  - [yuikonnu - Yume no Mata Yume \[Hard\]](https://osu.ppy.sh/beatmapsets/812160#osu/1703566)
  - [Dictate - Militant \[Hard\]](https://osu.ppy.sh/beatmapsets/785677#osu/1649483)
- ForceMod
  - [Ghost - From The Pinnacle To The Pit \[Insane\]](https://osu.ppy.sh/beatmapsets/425520#osu/919002)
  - [XL Project - Necro Fantasia \[Collab\]](https://osu.ppy.sh/beatmapsets/23077#osu/79266)
- Tiebreaker
  - **[Halozy - Paranoid Lost \[Soda's Lunatic\]](https://osu.ppy.sh/beatmapsets/440925#osu/964622)**

### Qualifiers

- NoMod
  - [FELT - Story \[Lunatic\]](https://osu.ppy.sh/beatmapsets/297409#osu/944709)
  - [Street - Sakura Fubuki \[Another\]](https://osu.ppy.sh/beatmapsets/362989#osu/827019)
  - [Sakuzyo - FAFNIR \[KKip's Insane\]](https://osu.ppy.sh/beatmapsets/777255#osu/1637690)
  - [Streetlight Manifesto - The Blonde Lead The Blind \[Insane\]](https://osu.ppy.sh/beatmapsets/376545#osu/826347)
- Hidden
  - [Saratoga (CV: Uesaka Sumire) - Souzetsu Gekkou \[Insane\]](https://osu.ppy.sh/beatmapsets/765452#osu/2178006)
  - [Veil of Maya - Mikasa \[Insane\]](https://osu.ppy.sh/beatmapsets/480298#osu/1219509)
- HardRock

>   - [Celldweller - Good L*ck (Yo*'re F_cked) \[Hard\]](https://osu.ppy.sh/beatmapsets/1069112#osu/2239676)

Reviewer 2: ```suggestion
  - [Celldweller - Good L\*ck (Yo\*'re F\_cked) \[Hard\]](https://osu.ppy.sh/beatmapsets/1069112#osu/2239676)
```

  - [Zektbach - meme \[Hard\]](https://osu.ppy.sh/beatmapsets/68617#osu/199410)
- DoubleTime
  - [senya - Terasareru kurai no Shiawase \[Collab Hard\]](https://osu.ppy.sh/beatmapsets/747507#osu/1575098)
  - [Creo - Rivals \[deetz' Hard\]](https://osu.ppy.sh/beatmapsets/1012878#osu/2142929)

---

## Match results

### Grand Finals

> |            Blue team |       |     | Red team  | Match link                                          |

Reviewer 2: ```suggestion
| Team 1 |  |  | Team 2 | Match link |
```
Please use this format instead

> | -------------------: | :---: | :-: | :-------- | :-------------------------------------------------- |

Reviewer 2: ```suggestion
| --: | :-: | :-: | :-- | :-- |
```

| **Norank MaMa <3 U** | **7** |  3  | Uprankers | [#1](https://osu.ppy.sh/community/matches/63280403) |

### Losers Finals

|     Blue team |     |     | Red team    | Match link       |
| ------------: | :-: | :-: | :---------- | :--------------- |

> | **Uprankers** |  0  | -1  | Salted_Fish | _win by default_ |

Reviewer 2: ```suggestion
| **Uprankers** |  0  | -1  | Salted_Fish | *win by default* |
```
Use asterisks instead of underscores, apply for; win by defaults, nullified

- Low roll: 1 ban
- High roll: 2 bans
- Low roll: 1 ban

Modpools cannot be fully banned. At least one map has to be available in each modpool

After the ban phase, the team will play maps with alternating picks until one team wins enough points to win a match.

1. The tiebreaker cannot be picked in a normal game scenario.
2. ForceMod maps are limited to: HD, HR, FL, EZ. You can stack these mods for a higher risk-higher reward scenario. With ForceMod at least one player has to pick a mod.
3. If both of the teams draw the map, then that map will not be counted towards the final score. However, it will be recorded for statistical purposes.
4. The tiebreaker is Free Mod, which means you can pick the mods from rule B or go NoMod

### Rescheduling Instructions

The schedule will be released 6 days in advance of the first match, usually the Monday before the weekend, on Google Calendar.

- A reschedule will only be approved if submitted 24 hours in advance.
- DM the team captain for the reschedule and figure out a new date and time with them
- In the reschedule channel tag Deadpulse or Kristoffer09 with a screenshot of the conversation so they can update the schedule.

> [flag_ca]: /wiki/shared/flag/CA.gif "Canada"

Reviewer 2: ```suggestion
[flag_CA]: /wiki/shared/flag/CA.gif "Canada"
```
Uppercase all flag references
